src/EnergyPlus/DualDuct.cc

```diff
@@ -2351,12 +2057,13 @@ namespace DualDuct {
                 Array1D_string cNumericFields(2);     // Numeric field names
                 Array1D_bool lAlphaBlanks(7, true);   // Logical array, alpha field input BLANK = .TRUE.
                 Array1D_bool lNumericBlanks(2, true); // Logical array, numeric field input BLANK = .TRUE.
```

Comment:
Local dynamic arrays that happen to be fixed-size ... tsk tsk tsk (I know nothing can be done about this because of InputProcessor, just pointing out).